cot/src/project.rs

```diff
@@ -1907,6 +1948,28 @@ impl<S: BootstrapPhase<AuthBackend = Arc<dyn AuthBackend>>> ProjectContext<S> {
     }
 }
 
+#[cfg(feature = "email")]
+impl<S: BootstrapPhase<Email = Arc<Email>>> ProjectContext<S> {
+    #[must_use]
+    /// Returns the email service for the project.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use cot::request::{Request, RequestExt};
+    /// use cot::response::Response;
+    ///
+    /// async fn index(request: Request) -> cot::Result<Response> {
+    ///     let email = request.context().email();
+    ///     // ...
+    /// #    unimplemented!()
+    /// }
+    /// ```
+    pub fn email(&self) -> &Arc<Email> {
```

Comment:
I think this should return `&Email` rather than `&Arc<Email>`. `Arc` should be an implementation detail that ideally shouldn't be exposed in public APIs, and since `Email` is essentially just a wrapper over another `Arc`, this seems redundant anyway.